bay-services/gremlin.yaml

```diff
@@ -1,5 +1,6 @@
 services:
-- hash: fe8e3b821c1ee0b1de02b589c992e3df36d55f0c
+- &gremlin_def
+  hash: fe8e3b821c1ee0b1de02b589c992e3df36d55f0c
```

Comment:
Updating hash to `ee802a798dff46fb6b507155d74717ef9ac8c82a` should fix the CI issue. But it also means we will promote a new image to production.

Reply:
Yes, got it now. I have updated the  hash.